src/main/java/ch/njol/skript/expressions/ExprName.java

```diff
@@ -70,6 +69,14 @@
 public class ExprName extends SimplePropertyExpression<Object, String> {
 	
 	private static final boolean inventoryTitles = Skript.methodExists(Inventory.class, "getTitle");
+	@SuppressWarnings("null")
+	private static Method getTitle;

```

Comment:
```suggestion
	private static final Method TITLE_METHOD;
```

Reply:
Ill make this change

Reply:
Okay, Its not letting me add `final`, when I do so, its making me initialize it. If I initialize it (with the reflection method) it makes me put it in a try/catch. If I initialize it as null, it won't let me assign it in the try catch as its final.